File: EMS.Web/EMSAngular/src/app/employee/employee.ts
export interface IEmployee
{
    id : number;
    firstName : string;
    lastName : string;
    department : Department;
    salary : number;
}

export class Employee implements IEmployee
{
    id : number =0;
    firstName : string =null;
    lastName : string =null;
    department : Department;
    salary : number = 0;
}

export enum Department
{
    Admin,
    Marketing,
    Development,
    Finance,
    HR
}

File: ESM.BL/Employee.cs
using System;

namespace ESM.BL
{
    public class Employee
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public Department Department { get; set; }
        public double Salary { get; set; }
    }
}
